src/main/java/mods/railcraft/common/carts/MinecartHooks.java

```diff
@@ -206,18 +206,17 @@ public void onEntityCollision(EntityMinecart cart, Entity other) {
     public AxisAlignedBB getCollisionBox(EntityMinecart cart, Entity other) {
         if (other instanceof EntityItem && RailcraftConfig.doCartsCollideWithItems())
             return other.getEntityBoundingBox();
-        if (other instanceof EntityPlayer)
-            return other.canBePushed() ? other.getEntityBoundingBox() : null; //            return other.boundingBox.contract(COLLISION_EXPANSION, 0, COLLISION_EXPANSION);
-        return null;
+        return other.canBePushed() ? other.getEntityBoundingBox() : null; //            return other.boundingBox.contract(COLLISION_EXPANSION, 0, COLLISION_EXPANSION);
```

Comment:
This allows any entity to push minecarts yes? Probably not desirable.

Reply:
Hmm, I think I added that in order to fix the locomotive hitting thing.